swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala

```diff
@@ -420,7 +421,7 @@ class SwaggerSpec2 extends ScalatraSpec with JsonMatchers {
     val JObject(definitions) = j
     definitions.map {
       case (name, definition) =>
-        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "in", "flow", "authorizationUrl", "scopes")
+        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "description", "in", "flow", "authorizationUrl", "scopes")
```

Comment:
Perhaps you should add "description" field to `verifyAuthorizations()` as well.
https://github.com/scalatra/scalatra/blob/3af64ba1c2011db5f760253958b5c3bf0c76424e/swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala#L129